pkg/services/search/service.go

```diff
@@ -11,11 +11,7 @@ import (
 )
 
 func init() {
-	registry.Register(&registry.Descriptor{
-		Name:         "SearchService",
-		Instance:     &SearchService{},
-		InitPriority: 20,
-	})
+	registry.RegisterService(&SearchService{})
```

Comment:
This had priority 20 before and default prio was low so think that using medium should be fine. Found another service having search.SearchService as dependency and that service was registered with priority low.